### ISSUE TITLE
For discussion: add a timestamped model to models

### ISSUE THREAD

**Issue by rshorey** [newsdev/int-newsapps-template#5]
really, when don't we want `created` and `updated`? I propose we subclass model in `models.py` here to make that easy. @jeremyjbowers @smbsimon, thoughts?

**Comment by jeremyjbowers** [comment]
I use a `BaseModel` class all the time that has `created`, `updated`, and `active`, which is a boolean.

Django:
```
class BaseModel(models.Model):
    active = models.BooleanField(default=True)
    created = models.DateTimeField(default=timezone.now)
    updated = models.DateTimeField(auto_now=True)

    class Meta:
        abstract = True

    def __unicode__(self):
        return self.__str__()
```

Peewee/Flask:
```
class BaseModel(Model):
    active = BooleanField(default=True)
    created = DateTimeField(default=datetime.datetime.now())
    updated = DateTimeField(default=datetime.datetime.now())
    
    class Meta:
        database = database
```

**Comment by rshorey** [comment]
resolved by 3b51865810eb80896ad454b3ceb40e14924b471b
